styleguide/global/global.js

```diff
@@ -49,16 +49,20 @@ jQuery(document).ready(($) => {
 
     // Callback to check if inbounds of side menu
     function checkInBounds(wt) {
+      // wt = pixels from top of window to top of scrolled area
       const filterFormHeight = parseInt($filterForm.outerHeight(), 10);
       if (!inBounds) {
         $stickyItemTop = parseInt($filterForm.offset().top, 10);
       }
+      // check if sticky form is scrolled passed top nav
       const windowTopInBounds = $stickyItemTop - $mainNavHeight < wt;
+      // get bottom offset of side bar
       const filterBottom = parseInt($filterBar.outerHeight(), 10) +
                            parseInt($filterBar.offset().top, 10);
-      const formBottom = wt + filterFormHeight;
-      const filterGap = filterBottom + $mainNavHeight;
-      const filterInBounds = (filterGap - filterFormHeight) * 3 > formBottom;
+      // top val of scrolled window + height of form (not bar)
+      const formBottom = wt + filterFormHeight + $mainNavHeight;
+      // if bottom of fixed form is not passed bottom of filter side bar
+      const filterInBounds = formBottom < filterBottom;
```

Comment:
Since we're effectively changing this formula and we have the facilities in zoolander to do so now, we should probably implement some unit testing around this to cover all the various cases.